migration/FromMySqlToPostgreSql/MapDataTypes.php

```diff
@@ -164,7 +164,7 @@ private function __construct()
 
         'point' => [
             'increased_size'           => '',
```

Comment:
Why? <br>
While <code>geometry</code> is capable to hold point value, the 
<code>point</code> simply restrict a value to a specific type...